### PR TITLE
new version: dotpdnllc.paintdotnet 5.0.6

### PR DESCRIPTION
**Pull request by lychichem**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108092)

### CODE REVIEW

File: manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml [hunk 1]
```diff
@@ -25,13 +25,13 @@ FileExtensions:
...
+ReleaseDate: 2023-05-25
 Installers:
 - Architecture: x64
   NestedInstallerType: burn
   NestedInstallerFiles:
-  - RelativeFilePath: paint.net.5.0.5.install.x64.exe
-  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.5/paint.net.5.0.5.install.x64.zip
+  - RelativeFilePath: paint.net.5.0.6.install.x64.exe
+  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.6/paint.net.5.0.6.install.x64.zip
   InstallerSha256: D29723D636556D6EBF6E89EFF925F5BFF27654DAE388D9A9086F909BA71BB6E9
```
Comment (rodrigoborgesdeoliveira) [inline manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml:35]:
```suggestion
  InstallerSha256: 5631D9767BA74A5069099E89AE5CC60D54D561E288C162926663C17E48F5BE02
```

File: manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml [hunk 2]
```diff
@@ -41,8 +41,8 @@ Installers:
 - Architecture: arm64
   NestedInstallerType: burn
   NestedInstallerFiles:
-  - RelativeFilePath: paint.net.5.0.5.install.arm64.exe
-  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.5/paint.net.5.0.5.install.arm64.zip
+  - RelativeFilePath: paint.net.5.0.6.install.arm64.exe
+  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.6/paint.net.5.0.6.install.arm64.zip
   InstallerSha256: 39DBCEDFF1D63031DD3CDF1A76245087308795A46DB6A4389CB4A2C596C54B77
```
Comment (rodrigoborgesdeoliveira) [inline manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml:46]:
```suggestion
  InstallerSha256: A1DD74264094BD8AD44BB6CDBBD1A105265D8BEFF46BBB360AD65284B050DCFE
```

File: manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml [hunk 3]
```diff
@@ -52,17 +52,17 @@ Installers:
 - Architecture: x64
   NestedInstallerType: wix
   NestedInstallerFiles:
-  - RelativeFilePath: paint.net.5.0.5.winmsi.x64.msi
-  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.5/paint.net.5.0.5.winmsi.x64.zip
+  - RelativeFilePath: paint.net.5.0.6.winmsi.x64.msi
+  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.6/paint.net.5.0.6.winmsi.x64.zip
   InstallerSha256: 45001F036B6F8A72997F696B167C473F3E60BD1E0F22756C246EFD26D3478004
```
Comment (rodrigoborgesdeoliveira) [inline manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml:57]:
```suggestion
  InstallerSha256: 9E2228363752EA2D65276B437B9B34EDB4F9626A1DDE867B985FE0C4D22C2264
```

File: manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml [hunk 4]
```diff
@@ -52,17 +52,17 @@ Installers:
...
   InstallerSwitches:
     InstallLocation: INSTALLDIR="<INSTALLPATH>"
 - Architecture: arm64
   NestedInstallerType: wix
   NestedInstallerFiles:
-  - RelativeFilePath: paint.net.5.0.5.winmsi.arm64.msi
-  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.5/paint.net.5.0.5.winmsi.arm64.zip
+  - RelativeFilePath: paint.net.5.0.6.winmsi.arm64.msi
+  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.6/paint.net.5.0.6.winmsi.arm64.zip
   InstallerSha256: 2AB279E52580E7EFCFF6B0BD2F7AC807D7B0B5356EC2480AC92AB821F67EA62B
```
Comment (rodrigoborgesdeoliveira) [inline manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml:66]:
```suggestion
  InstallerSha256: 23892BF328738D232E5F0EC52FA57421D62616A3BAC0D27DE4B7072543B96BEC
```

File: manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml [hunk 5]
```diff
@@ -72,14 +72,14 @@ Installers:
   NestedInstallerFiles:
   - RelativeFilePath: paintdotnet.exe
     PortableCommandAlias: paint.net
-  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.5/paint.net.5.0.5.portable.x64.zip
+  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.6/paint.net.5.0.6.portable.x64.zip
   InstallerSha256: 0C7B0B39AEBF989AD1CB2ECC9A32DC60A1BE3192A5266EEBF99047FB67FF0C55
```
Comment (rodrigoborgesdeoliveira) [inline manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml:76]:
```suggestion
  InstallerSha256: C236ABBFD5CB2EF5E14E274417CDA5FD771AD1BF72F5DE21993FE88CF4B035D2
```

File: manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml [hunk 6]
```diff
@@ -72,14 +72,14 @@ Installers:
...
 - Architecture: arm64
   NestedInstallerType: portable
   NestedInstallerFiles:
   - RelativeFilePath: paintdotnet.exe
     PortableCommandAlias: paint.net
-  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.5/paint.net.5.0.5.portable.arm64.zip
+  InstallerUrl: https://github.com/paintdotnet/release/releases/download/v5.0.6/paint.net.5.0.6.portable.arm64.zip
   InstallerSha256: AB3D9092210E18E327E55839C67157B31EE65C68AB6B2E0055B5BFDD7A33D731
```
Comment (rodrigoborgesdeoliveira) [inline manifests/d/dotPDNLLC/paintdotnet/5.0.6/dotPDNLLC.paintdotnet.installer.yaml:83]:
```suggestion
  InstallerSha256: DC83BF753510CC68D976764F499A3D63490532221E717C0B3FAE6B5DCBE8EA46
```

**Review by rodrigoborgesdeoliveira — CHANGES_REQUESTED** [review]
Hi @lychichem, in case it helps, I left the actual hash of each file. 🙂

**Review by rodrigoborgesdeoliveira — CHANGES_REQUESTED** [review]
Hi @lychichem, in case it helps, I left the actual hash of each file. 🙂

**Comment by lychichem** [comment]
> Hi @lychichem, in case it helps, I left the actual hash of each file. 🙂 

I know you are in hurry, bit please be patient😂I just don't want to download all the installer and want the bot tell me what the actual hash are.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**